### DISCUSSION TITLE
load_archive and model from_archive

### DISCUSSION THREAD

**Discussion by vikigenius** [allenai/allennlp#5623] [Q&A]
Looking at this example: https://github.com/allenai/allennlp/blob/f877fdc30d18178b88c335fbd92722fb77c42d93/test_fixtures/basic_classifier/experiment_from_archive.jsonnet#L8-L11

When I use this for training, it produces an archive where during inference time when I try to use to load_archive it needs the original archive to be there as well.

Can this be avoided?

I basically want to further train a model from a previous archive a1 using new data. So I use the above snippet.
But this produces an archive a2 where the config.json mentions a1. Which means during inference/prediction time I need both a1 and a2 archives in my directory structure. When technically I shouldn't be needing a1.

How do I avoid this dependency on a1 without manually modifying the a2 archive to change the model definition.

**Comment by epwalsh** [comment]
Hey @vikigenius, see https://github.com/allenai/allennlp/issues/5604#issuecomment-1079451065.

Long story short, the easiest way around this is to manually modify the `config.json` file within the `model.tar.gz` to remove any references to the old archive.
